graylog2-web-interface/src/views/components/widgets/EditWidgetFrame.tsx

```diff
@@ -93,7 +99,7 @@ const EditWidgetFrame = ({ children }: Props) => {
            animation={false}
            dialogComponentClass={EditWidgetDialog}
            enforceFocus={false}>
-      <SearchBarForm initialValues={{ limitDuration: 0, timerange, streams, queryString }}
+      <SearchBarForm initialValues={{ limitDuration, timerange, streams, queryString }}
```

Comment:
It's not really part of the entire state, and I agree having a wrapping context may be better, but this is already written in such a way across Search and Dashboard, so it would require updating in a few places. Maybe this is something we can include as a follow-up later or do you feel it is important enough to rework all of them?

Reply:
If you keep track of it and rework it before the picker is merged into `master`, then this is fine.

Reply:
This conversation was followed-up in https://github.com/Graylog2/graylog2-server/pull/9893